### PR TITLE
Add Gerardo and Joshua as the owner of the Prow config files

### PR DESCRIPTION
**Pull request by chizhg**
/cc @albertomilan @joshua-bone @gerardo-lc

### CODE REVIEW

**Review by joshua-bone — APPROVED** [review]

File: config/prod/prow/OWNERS [hunk 1]
```diff
@@ -1,8 +1,10 @@
 approvers:
```
Comment (n3wscott) [inline config/prod/prow/OWNERS:1]: I am not sure this file will do much in the near future. We are moving to CODEOWNERS.

Comment (chizhg) [reply config/prod/prow/OWNERS:1]: Yeah I can see that.. I'll close this PR then. Will try to bring it up again if needed after we move to CODEOWNERS.

**Pull request closed without merging**